lua/plugins/service.lua

```diff
@@ -8,6 +8,23 @@ local plugin = {}
 
 function plugin.onTextMessage(msg, blocks)
 
+		if blocks[1] == 'new_chat_member' then
+    local status = db:hget(('chat:%d:settings'):format(msg.chat.id), 'Clean_Msg_Service') or config.chat_settings.settings.Clean_Msg_Service
+    if status == 'on' then
+		if msg.service then
+			api.deleteMessage(msg.chat.id, msg.message_id)
+			end
+		end
+	end
+	if blocks[1] == 'left_chat_member' then
+    local status = db:hget(('chat:%d:settings'):format(msg.chat.id), 'Clean_Msg_Service') or config.chat_settings.settings.Clean_Msg_Service
```

Comment:
same as above, but also note that you have not set a default value for your option in config.lua

Reply:
I already change this, I added it in the last update of the code